README.md

```diff
@@ -1,170 +0,0 @@
-# Theatre [![CircleCI](https://circleci.com/gh/gocardless/theatre.svg?style=svg)](https://circleci.com/gh/gocardless/theatre)
```

Comment:
Are we meant to delete the readme?

Reply:
We were planning on adding them all back at the end and ensuring they are all up to date and contain everything we want.